python/GafferOSL/FocalBlur.gfr

```diff
@@ -0,0 +1,936 @@
+__children["FocalBlur"]["CustomCameraExpression"]["__in"]["p3"].setInput( __children["FocalBlur"]["focusDistance"] )
+__children["FocalBlur"]["CustomCameraExpression"]["__in"]["p4"].setInput( __children["FocalBlur"]["horizontalAperture"] )
+__children["FocalBlur"]["CustomCameraExpression"]["__uiPosition"].setValue( imath.V2f( 765.393494, 21.787466 ) )
+__children["FocalBlur"]["NoCameraExpression"]["__in"]["p0"].setInput( __children["FocalBlur"]["focusDistance"] )
+__children["FocalBlur"]["NoCameraExpression"]["__uiPosition"].setValue( imath.V2f( 762.390442, 21.7854328 ) )
+__children["FocalBlur"]["Dot25"]["in"].setInput( __children["FocalBlur"]["Dot12"]["out"] )
+__children["FocalBlur"]["Dot25"]["__uiPosition"].setValue( imath.V2f( 822.9505, 85.5897293 ) )
+__children["FocalBlur"]["Dot26"]["in"].setInput( __children["FocalBlur"]["Dot25"]["out"] )
+__children["FocalBlur"]["Dot26"]["__uiPosition"].setValue( imath.V2f( 822.9505, 35.2791519 ) )
+Gaffer.Metadata.registerValue( __children["FocalBlur"]["cameraPath"], 'nodule:type', '' )
+Gaffer.Metadata.registerValue( __children["FocalBlur"]["cameraPath"], 'stringPlugValueWidget:placeholderText', 'Render Camera' )
+Gaffer.Metadata.registerValue( __children["FocalBlur"]["cameraPath"], 'layout:section', 'Settings' )
+Gaffer.Metadata.registerValue( __children["FocalBlur"]["cameraPath"], 'layout:index', 7 )
+Gaffer.Metadata.registerValue( __children["FocalBlur"]["cameraPath"], 'layout:visibilityActivator', 'cameraScene' )
+Gaffer.Metadata.registerValue( __children["FocalBlur"]["cameraPath"], 'description', 'The scene path to the camera in `cameraScene` that you want to take settings from. If unset, defaults to the render camera set in the scene globals.' )
+Gaffer.Metadata.registerValue( __children["FocalBlur"]["cameraPath"], 'plugValueWidget:type', 'GafferSceneUI.ScenePathPlugValueWidget' )
+Gaffer.Metadata.registerValue( __children["FocalBlur"]["cameraPath"], 'scenePathPlugValueWidget:scene', 'cameraScene' )
+Gaffer.Metadata.registerValue( __children["FocalBlur"]["cameraPath"], 'scenePathPlugValueWidget:setNames', IECore.StringVectorData( [ '__cameras' ] ) )
+Gaffer.Metadata.registerValue( __children["FocalBlur"]["cameraPath"], 'scenePathPlugValueWidget:setsLabel', 'Show only cameras' )
+__children["FocalBlur"]["Expression"]["__in"]["p0"].setInput( __children["FocalBlur"]["cameraPath"] )
+__children["FocalBlur"]["Expression"]["__uiPosition"].setValue( imath.V2f( 775.942566, 29.858284 ) )
+Gaffer.Metadata.registerValue( __children["FocalBlur"]["depthChannel"], 'nodule:type', '' )
+Gaffer.Metadata.registerValue( __children["FocalBlur"]["depthChannel"], 'layout:section', 'Depth' )
+Gaffer.Metadata.registerValue( __children["FocalBlur"]["depthChannel"], 'layout:index', 18 )
+Gaffer.Metadata.registerValue( __children["FocalBlur"]["depthChannel"], 'description', 'Which image channel to use as the depth when computing the amount of blur.' )
```

Comment:
Would be user-friendly to make use of a `GafferImageUI.ChannelPlugValueWidget` for `depthChannel`.